docs/examples/driver_examples/Qcodes example with Minicircuits Switch boxes (USB-XSPDT).ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Example for Minicircuits Switch Boxes controlled via USB"
+    "## QCoDeS Example with Minicircuits Switch Boxes Controlled via USB"
```

Comment:
Should this be a level 1 heading?

Reply:
On the local html it shows up as level 1 so I missed it. Corrected it now, should I make a new commit for it? Or I can commit is with some other doc improves.

Reply:
Its best to make a new commit on this branch and then push